lib/saml/user_attributes/ssoe.rb

```diff
@@ -53,7 +54,9 @@ def birth_date
       end
 
       def email
-        safe_attr('va_eauth_emailaddress')
+        val = safe_attr('va_eauth_emailaddress')
+        val ||= safe_attr('va_eauth_commonname')
```

Comment:
I'm surprised by  this line. I know you have the guard on the next line to check if it's an email address. And I know one of the captured payloads for inbound SSO had an email address here but I'm not 100% confident that that represents a user-supplied email address (vs. e.g. coercing an MHV username into an email address with a phony domain) We should validate that  with an additional inbound MHV user SAML payload.

Reply:
I only made that change because of the value I saw in the `ssoe_inbound_mhv_premium` factory.  I'll remove the changes if thats not guarenteed to be a verfied email